x-pack/test/alerting_api_integration/common/config.ts

```diff
@@ -100,6 +100,27 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
                 xyzSecret2: 'credential2',
               },
             },
+            {
```

Comment:
More pre-configured actions, used in the tests below.  Show up in the various `getAll()` tests as well ...